camel/configs/gemini_config.py

```diff
@@ -0,0 +1,138 @@
+                max_output_tokens:
+                    The maximum number of tokens to include in a
+                    candidate.
+
+                    If unset, this will default to output_token_limit specified
+                    in the model's specification.
+                temperature:
+                    Controls the randomness of the output. Note: The
+                    default value varies by model, see the `Model.temperature`
+                    attribute of the `Model` returned the `genai.get_model`
+                    function.
+
+                    Values can range from [0.0,1.0], inclusive. A value closer
+                    to 1.0 will produce responses that are more varied and
+                    creative, while a value closer to 0.0 will typically result
+                    in more straightforward responses from the model.
+                top_p:
+                    Optional. The maximum cumulative probability of tokens to
+                    consider when sampling.
+
+                    The model uses combined Top-k and nucleus sampling.
+
+                    Tokens are sorted based on their assigned probabilities so
+                    that only the most likely tokens are considered. Top-k
+                    sampling directly limits the maximum number of tokens to
+                    consider, while Nucleus sampling limits number of tokens
+                    based on the cumulative probability.
+
+                    Note: The default value varies by model, see the
+                    `Model.top_p` attribute of the `Model` returned the
+                    `genai.get_model` function.
+
+                top_k (int):
+                    Optional. The maximum number of tokens to consider when
+                    sampling.
+
+                    The model uses combined Top-k and nucleus sampling.
+
+                    Top-k sampling considers the set of `top_k` most probable
+                    tokens. Defaults to 40.
+
+                    Note: The default value varies by model, see the
+                    `Model.top_k` attribute of the `Model` returned the
+                    `genai.get_model` function.
+
+                response_mime_type:
+                    Optional.
+                    Output response mimetype of the generated candidate text.
+
+                    Supported mimetype:
+                        `text/plain`: (default) Text output.
+                        `application/json`: JSON response in the candidates.
+
+                response_schema:
+                    Optional. Specifies the format of the JSON requested
+                    if response_mime_type is `application/json`.
+
+            safety_settings: Overrides for the model's safety settings.
+            stream: If True, yield response chunks as they are generated.
+            tools: `protos.Tools` more info coming soon.
+            request_options: Options for the request.
```

Comment:
The order of the docstring is not the same as the field. It is better to make them the same order